### ISSUE TITLE
Allow interfaces to be rendered without a parent <label> field

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/app#87]
As of right now, each interface on the edit view gets rendered with a corresponding `<label>` for `a11y` reasons. However, certain interfaces will include their own `<label>`s. This leads to invalid html. For example checkboxes:

```html
<label>
  Checkboxes
  <label><input /> Checkbox A</label>
  <label><input /> Checkbox B</label>
</label>
```

Ideally, the interfaces have a way to let the application know _not_ to render the parent label, or maybe swap it out for a `<fieldset>` / `<legend>` combo. This is probably going to be a key in `meta.json`. 

Any smart ideas for a name @benhaynes? `containsLabel`, `noLabel`, `fieldset`

**Comment by benhaynes** [comment]
`labelOverride`? `overrideLabel`?

**Comment by benhaynes** [comment]
`supressLabel`?

**Comment by rijkvanzanten** [comment]
Maybe I could scale up the option to let it set the parent markup:

`"parentElement": "fieldset"`

Would be very flexible, not sure if we would need that flexibility though 🤔

**Comment by benhaynes** [comment]
Hmm, might be too much control – but we could do that. It would give more control over where the label and comment go – but we want to make sure the styling of those elements can't be changed (and they must be there).

**Comment by rijkvanzanten** [comment]
True! I think it's going to be a boolean that sets the difference between:

```html
<label>
  label
  <slot />
</label>
```

and 

```html
<fieldset>
  <legend>label</legend>
  <slot />
</fieldset>
```

**Comment by rijkvanzanten** [comment]
This has been implemented in #107
